manifests/base/imagestream.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: aidevsecops-tutorial
+spec:
+  lookupPolicy:
+    local: true
```

Comment:
this imag stream has no tag, so there is really now image in here...

Reply:
I was not sure if we wanted to have `overlays/`! Thanks for review, I added it :) there is no image on quay yet in any case!

/hold

Reply:
we should let `base/imagestream.yaml` have no tags and include an overlay `test` or something, include imagestream with a tag in that. 
In this way, we would be demonstrating a good practice of kustomize base/overlays dir.